internal/bminventory/inventory.go

```diff
@@ -4050,16 +4042,13 @@ func (b *bareMetalInventory) DisableHost(ctx context.Context, params installer.D
 			return common.NewApiError(http.StatusNotFound, err)
 		}
 		log.WithError(err).Errorf("failed to get host %s", params.HostID)
-		msg := fmt.Sprintf("Failed to disable host %s: error fetching host from DB", params.HostID.String())
-		b.eventsHandler.AddEvent(ctx, params.ClusterID, &params.HostID, models.EventSeverityError, msg, time.Now())
+		eventgen.SendDisableHostFailedFetchEvent(ctx, b.eventsHandler, &params.ClusterID, params.HostID, params.ClusterID)
```

Comment:
@arielireni this is a V1 function ( we don't support DisableHost in V2), so you can assume that host's InfraEnvID  = ClusterID

Reply:
If this will never have an actual `infra_env_id` (as there is no support for this action in V2 API), why do we have such field in the event definition?
Perhaps we need to remove it, or change it to `UUID_PTR` and pass a `nil` here.

Reply:
@masayag agreed.